sinonjs/referee
# Update insecure packages as reported by npm audit

This PR updates dependencies to stop using known insecure packages from npm

#### How to verify - mandatory
1. Check out `master`
1. `npm install`
1. `npm audit`
1. Observe several warnings
1. Check out this branch
1. `npm install`
1. `npm audit`
1. Observe there are no warnings
